Makefile

```
@@ -28,7 +28,7 @@ help:
 
 init-db: compose-build
 	@echo "Check for new migrations to run..."
-	@docker-compose run --rm $(IMAGE_NAME) \
+	@docker-compose run --rm -e ANALYTICS_OPT_OUT \ $(IMAGE_NAME) \
```

No `\` needed here, since you're not adding a newline.
```suggestion
	@docker-compose run --rm -e ANALYTICS_OPT_OUT $(IMAGE_NAME) \
```